### PR TITLE
Added Intros to REST APIs Pages

### LINKED ISSUE #6976: Add intros to REST API pages
### What article on docs.github.com is affected?

- "[Markdown](https://docs.github.com/en/rest/reference/markdown)"
- "[Meta](https://docs.github.com/en/rest/reference/meta)"
- "[Migrations](https://docs.github.com/en/rest/reference/migrations)"
- "[Organizations](https://docs.github.com/en/rest/reference/orgs)"
- "[Packages](https://docs.github.com/en/rest/reference/packages)"

### What part(s) of the article would you like to see updated?

Each of these articles needs an intro to help orient users to the content on the page and what each API can be used for. Intros should be one sentence long and define what the feature is.

For "[Packages](https://docs.github.com/en/rest/reference/packages)," you can cut and paste the first sentence in the article for the intro.

For "[Markdown](https://docs.github.com/en/rest/reference/markdown)," "[Meta](https://docs.github.com/en/rest/reference/meta)," "[Migrations](https://docs.github.com/en/rest/reference/migrations)," and "[Organizations](https://docs.github.com/en/rest/reference/orgs)" you will need to write new intros.

For more information about writing intros, see:

- [Information about writing intros](https://github.com/github/docs/blob/main/contributing/content-model.md#intro) in the GitHub docs content model
- [Information about the `intro` frontmatter property](https://github.com/github/docs/tree/main/content#intro) in the GitHub Docs content directory README

### Additional information

This issue is for the Campus Experts + Docs program and not currently available for contributors outside the program to work on.

### PR DESCRIPTION
**Pull request by codewithdev**
### Why:

Closes #6976 

### What's being changed:

Intro added for the following REST API Pages,
 - [Migrations](https://docs.github.com/en/rest/reference/markdown)
 - [Meta](https://docs.github.com/en/rest/reference/meta)
 - [Migrations](https://docs.github.com/en/rest/reference/migrations)
 - [Organizations](https://docs.github.com/en/rest/reference/orgs)
 - [Packages](https://docs.github.com/en/rest/reference/packages)

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@codewithdev Thank you for this PR 👍  I will triage it for review ⚡

**Review by tranhieu0807 — DISMISSED** [review]

**Review by felicitymay — COMMENTED** [review]
@codewithdev, nice to see you again 👋🏻 

Thanks for taking the time to research these API articles, work out a summary of what they do, and write an introduction. 👍🏻  

I've made some small suggestions and will commit those before merging your introductions. Thanks for your help ✨

**Review by felicitymay — DISMISSED** [review]

File: content/rest/reference/markdown.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Markdown
+intro: 'The Markdown API enables you to render a markdown document as an HTML page or raw text'
```
Comment (felicitymay) [inline content/rest/reference/markdown.md:3]: Sorry - my error.

```suggestion
intro: 'The Markdown API enables you to render a markdown document as an HTML page or as raw text.'
```

**Review by felicitymay — APPROVED** [review]

**Comment by Oleksandr3003** [comment]
Учреждение банка:
ПриватБанк

МФО банка:
305299

Получатель платежа
ГОПАЙЦА АЛЕКСАНДР ВАСИЛЬЕВИЧ

IBAN:
UA513052990000026201733718591

Счет получателя:
26201733718591

Валюта карты:
UAH

РНУКПН получателя:
2810609011

Назначение платежа:
Пополнение счета ГОПАЙЦА АЛЕКСАНДР ВАСИЛЬЕВИЧУчреждение банка:
ПриватБанк

МФО банка:
305299

Получатель платежа
ГОПАЙЦА АЛЕКСАНДР ВАСИЛЬЕВИЧ

IBAN:
UA513052990000026201733718591

Счет получателя:
26201733718591

Валюта карты:
UAH

РНУКПН получателя:
2810609011

Назначение платежа:
Пополнение счета ГОПАЙЦА АЛЕКСАНДР ВАСИЛЬЕВИЧ

вт, 13 июл. 2021 г., 20:03 github-actions[bot] ***@***.***>:

**Pull request merged**